Formula/git.rb

```diff
@@ -96,7 +96,6 @@ def install
     ]
     args << "NO_OPENSSL=1" << "APPLE_COMMON_CRYPTO=1" if build.without? "brewed-openssl"
 
-    system "make", "test", *args
```

Comment:
instead of a full revert, what about add an `if build.bottle?` to trigger tests on binary release builds?